### ISSUE TITLE
Latest version errors out with "TypeError: 'NoneType' object is not iterable".

### ISSUE THREAD

**Issue by cg4242** [gene-git/dmarc_report#2]
System: Arch Linux
dmarc_report installed via "yay -S dmarc_report".
Python 3.12.7

```
Traceback (most recent call last):
  File "/usr/bin/dmarc-rpt", line 49, in <module>
    main()
  File "/usr/bin/dmarc-rpt", line 21, in main
    report = DmarcRpt()
             ^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/dmarc_report/lib/class_dmarc.py", line 161, in __init__
    self.dom_nets = cidrs_to_nets(self.opts.dom_ips)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/dmarc_report/lib/cidr.py", line 26, in cidrs_to_nets
    nets = [ipaddress.ip_network(cidr, strict=strict) for cidr in cidrs]
                                                                  ^^^^^
TypeError: 'NoneType' object is not iterable
```

**Comment by gene-git** [comment]
Thank you  - my apologies. 
I missed it as the tests were  run with ~/.config/dmarc_report/config containing : dom_ips = [x.x.x.x/n]

Fixed in tag 4.10.0

**Comment by gene-git** [comment]
All good now?
